### ISSUE TITLE
Illustration of conceptual model

### ISSUE THREAD

**Issue by jwyoung** [karlbenedict/agilecuration#1]
At least for obtuse folks like me it would help to note 'time' on the x axis of the lower diagram on effort.

**Comment by jwyoung** [comment]
Somewhere in the document I would describe what is meant by 'use metric' when used in the illustration. The description doesn't have to be part of the diagram and it doesn't have to be definitive but some possible examples (i.e. downloads, references in publications, etc.) would be helpful .

**Comment by karlbenedict** [comment]
Yep - adding time to the x-axis is good, though I'm working on an expansion of the figure that anchors the zero-time point at different points in time along the x-axis of the use-metric plot.

**Comment by karlbenedict** [comment]
Examples of use-metrics are indeed necessary, though specific metrics may very well vary depending upon the specific community of users that a particular support scenario is executed for ...

... data downloads for some researcher users
... metadata downloads for centralized discovery systems (e.g. Data.gov, DataONE)
... data service requests (e.g. OpenDAP, WCS, WFS, REST services)
... visualization services for data (e.g. WMS, graphs)

The different request/response patterns for these use patterns suggest that some sort of indexing for these mixed metrics is probably useful to handle scale differences between them.

**Comment by karlbenedict** [comment]
Need to integrate cumulative model for requirements met through multiple iterations for different user communities / uses.
